packages/suite/src/components/wallet/CoinjoinSummary/index.tsx

```diff
@@ -1,31 +1,38 @@
 import React from 'react';
 import styled from 'styled-components';
-import { useSelector } from '@suite-hooks';
-import { Account } from '@suite-common/wallet-types';
-import { CoinjoinSetup } from './CoinjoinSetup';
-import { CoinjoinSessionStatus } from './CoinjoinSessionStatus';
 
-const Wrapper = styled.div`
-    margin-bottom: 24px;
+import { H3 } from '@trezor/components';
+import { Translation } from '@suite-components/Translation';
+import { AnonymityIndicator } from '@wallet-components/Privacy/AnonymityIndicator';
```

Comment:
nit: Not sure about the `Privacy` naming, I would stick to the term `Coinjoin` or `PrivacyAccount`

Reply:
renamed to PrivacyAccount d1843400515412a4b05ed916f65fd3bd7cf86ef2